src/cascadia/TerminalSettingsEditor/Profiles.cpp

```diff
@@ -428,7 +444,8 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
     }
 
     Profiles::Profiles() :
-        _ColorSchemeList{ single_threaded_observable_vector<ColorScheme>() }
+        _ColorSchemeList{ single_threaded_observable_vector<ColorScheme>() },
+        _previewControl{ Control::TermControl(Model::TerminalSettings(), make<PreviewConnection>()) }
```

Comment:
```suggestion
        _previewControl{ Control::TermControl(Model::TerminalSettings{}, make<PreviewConnection>()) }
```
nit: `()` makes it ambiguous if you're calling a function or a ctor sometimes. Dustin scared me into this one ages ago hahaha